modules/scalacheck/test/src/weaver/scalacheck/PropertyDogFoodTest.scala

```diff
@@ -53,39 +53,43 @@ object PropertyDogFoodTest extends IOSuite {
       events <- dogfood.runSuite(Meta.ParallelChecks).map(_._2)
       _      <- expect(events.size == 1).failFast
     } yield {
-      expect(events.headOption.get.duration() < 10000)
+      expect(events.headOption.get.duration() < 100000)
     }
   }
 
-  // 5 checks with perPropertyParallelism = 1 sleeping 1 second each should take at least 5 seconds
+  test("Discarded counts should be accurate") { dogfood =>
```

Comment:
can you keep the original ordering of the tests ? arbitrary re-orders just add noise to the diff

Reply:
Fixed in https://github.com/disneystreaming/weaver-test/pull/631/commits/56d22e637300f12339a628a4646346ed28877813